packages/KIcon/icons/icn-plus.svg

```diff
@@ -1,4 +1,4 @@
-<svg viewBox="0 0 20 20" fill="none" xmlns="http://www.w3.org/2000/svg">
+<svg viewBox="0 0 20 20" width="16" height="16" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
This looks good. I looked around and tried the values in `khcp-ui` and I didn't see where this would cause issues.